openverse_catalog/dags/common/loader/loader.py

```diff
@@ -44,6 +44,7 @@ def load_from_s3(
     sql.load_s3_data_to_intermediate_table(
         postgres_conn_id, bucket, key, identifier, media_type
     )
-    sql.upsert_records_to_db_table(
+    # Returns record count
+    return sql.upsert_records_to_db_table(
```

Comment:
So Airflow automatically sends an xcom with a key of `return_value`? That's great to know 😅

Reply:
[It does!](https://github.com/WordPress/openverse-catalog/pull/369#discussion_r810315410) Not super obvious though haha